concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    for subtask in Subtask.objects.all():
+        try:
+            report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
```

Comment:
You're not running in a transaction. If the script fails in the middle of the operation the changes will not be roleld back.

Reply:
Done